ProjectSettings/ProjectVersion.txt

```diff
@@ -1 +1 @@
-m_EditorVersion: 2017.2.1f1
+m_EditorVersion: 2017.3.0f3
```

Comment:
Please revert editor version.

Reply:
Agreed, Editor version should stay at 2017.2 until 2017.3.x is approved for use